src/nile/starknet_cli/deploy_account.py

```diff
@@ -0,0 +1,119 @@
+"""Starknet CLI overrides."""
+import argparse
+from types import SimpleNamespace
+from typing import List, Optional
+
+from starkware.starknet.cli.starknet_cli import (
+    add_deploy_account_tx_arguments,
+    assert_tx_received,
+    compute_max_fee,
+    get_gateway_client,
+    need_simulate_tx,
+    simulate_or_estimate_fee,
+    validate_max_fee,
+)
+from starkware.starknet.services.api.gateway.transaction import DeployAccount
+
+from nile.common import QUERY_VERSION, TRANSACTION_VERSION, get_account_class_hash
+from nile.core.types.utils import get_counterfactual_address
```

Comment:
This is for the sake of keeping the interface of `execute_call`, but we don't need to keep up with starknet's base implementation. I used the starknet deploy_account method as a baseline just because was easier to maintain the API of the methods (to modify the execute_call interface as less as possible). But the important thing here is just the returned DeployAccount.